packages/authentication/test/unit/bindings.test.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from 'testlab';
+import * as http from 'http';
+import {Strategy} from '../..';
+import {getAuthenticatedUser} from '../..';
+import {supertest} from 'testlab';
+
+describe('Authentication Bindings', () => {
+  let strategy: Strategy;
+  let app: http.Server;
+  const mockUser: User = {id: 'mock-user', role: 'mock-role'};
+  interface User {
+    id: string;
+    role: string;
+  }
+
+  describe('getAuthenticatedUser()', () => {
```

Comment:
In my opinion, using method name as the test name is entirely appropriate here, because this is a unit-test focused on testing that very method in isolation.

I also agree that we shouldn't be using method names as test names in higher-level tests, most notably acceptance tests written in BDD style.